src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -119,6 +119,33 @@ public static void LibyaTimeZone()
             Assert.True(libyaLocalTime.Equals(expectResult), string.Format("Expected {0} and got {1}", expectResult, libyaLocalTime));
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsWindows))]
```

Comment:
Why does this need to be Windows specific?  Wondering if it can be written in a way that works on Linux when running on an ICU that contains data for the same time zone.

Reply:
I restricted this to Windows only for multiple reasons, first Windows zone id `Yukon Standard Time` cannot be used on Linux. Second the rules on Linux is very different than Windows. I added this test case mainly to test Windows edge cases. As I mentioned Windows has some limitation which cannot specify more than one transition per year while Linux data don't have this restriction. I am not expecting Yukon on Linux would be any different than any other zone which we already testing.